src/arrow_reader.rs

```diff
@@ -427,7 +428,10 @@ impl Stripe {
     ) -> Result<Self> {
         let footer = Arc::new(r.stripe_footer(stripe).clone());
 
-        let compression = r.metadata().postscript.compression();
+        let compression = Compression::from_proto(
+            r.metadata().postscript.compression(),
+            r.metadata().postscript.compression_block_size,
+        );
```

Comment:
Maybe eventually can store `Compression` directly as field in `FileMetadata` so don't need to derive each time